backend/tracim_backend/lib/mail_fetcher/email_fetcher.py

```diff
@@ -171,6 +170,8 @@ def find_key_from_mail_address(
                 cls, "pattern {} does not match email address {} ".format(pattern, mail_address)
             )
             return None
+        else:
+            return None
```

Comment:
replace these three lines by a

```python
    return None
```

outside the if statement

Reply:
done.